README.md

```diff
@@ -37,7 +37,7 @@ return [
     'bootstrap' => ['gii'],
     'modules' => [
         'gii' => [
-            '__class' => Yiisoft\Yii\Gii\Module::class,
+            '__class' => Yiisoft\Yii\Gii\Gii::class,
```

Comment:
Why is `Module` renamed to `Gii`?

Reply:
There are no modules in Yii 3

Reply:
Looks like I didn't follow the development too much :), but why is it configured in a `modules` section, then?

Reply:
Leftover...

Reply:
Will the above section be removed from the docs?

Is there an equivalent the modules in Yii3?

Reply:
> Is there an equivalent the modules in Yii3?

No. We may add these if needed but likely it won't be needed...

Reply:
> Yes, eventually. @rustamwin or is it better to remove it now?

Yes. I'll remove it in this PR